resources/js/beatmapsets-show/info.tsx

```diff
@@ -198,7 +198,7 @@ export default class Info extends React.Component<Props> {
                   <React.Fragment key={`${tag.name}-${i}`}>
                     <a
                       className='beatmapset-info__link'
-                      href={route('beatmapsets.index', { q: tag.name })}
+                      href={route('beatmapsets.index', { q: makeSearchQueryOption('tag', tag.name) })}
```

Comment:
the function initially has something like `esExact` parameter but then removed because not yet needed but I suppose it's needed now?

Or update the search library to remove the extra quotes when doing db lookup